patsicko/my-brand-bakend
# Ft blog retrieve single 007

## What this PR do
Retrieve single blog
## Description of the tasks to be completed
- GET /api/singleBlog
[Finishes#007]

## How should this be manually tested
- open postman
- set your HTTP request to GET
- input API endpoint
- Click Send.
- Status: 200 OK should be displayed
- Retrieved datum is showing up in the body enveloped in JSend object

## Trello story id
#007
